modules/raft/src/main/java/org/apache/ignite/internal/raft/Loza.java

```diff
@@ -96,11 +98,16 @@ public class Loza implements IgniteComponent {
      *
      * @param clusterNetSvc Cluster network service.
      * @param dataPath      Data path.
+     * @param clock         A hybrid logical clock.
      */
-    public Loza(ClusterService clusterNetSvc, Path dataPath) {
+    public Loza(ClusterService clusterNetSvc, Path dataPath, HybridClock clock) {
         this.clusterNetSvc = clusterNetSvc;
 
-        this.raftServer = new JraftServerImpl(clusterNetSvc, dataPath);
+        NodeOptions options = new NodeOptions();
+
+        options.setClock(clock);
```

Comment:
We already have this in service factory

Reply:
It is needed to pass clock to JraftServerImpl.
JraftServerImpl creates DefaultJRaftServiceFactory with the clock instance.

Reply:
I merged main into ignite3_tx and ignite-17221.
Now the clock is passed by NodeOptions.